app/main.go

```diff
@@ -62,8 +68,24 @@ func getIPAddress(r *http.Request) string {
 }
 
 func handlerPing(w http.ResponseWriter, r *http.Request) {
-	w.Write([]byte("pong"))
-	fmt.Println(time.Now(), getIPAddress(r), r.Method, r.RequestURI, r.UserAgent())
+	
+	requestBody, err := ioutil.ReadAll(r.Body)
+	n := rand.Intn(100)
+
+	<- time.After(time.Duration(lat) * time.Millisecond)
+	if n <= prob {
+		w.Write([]byte("pong"))
+		fmt.Println(time.Now(), getIPAddress(r), r.Method, r.RequestURI, r.UserAgent())
+	} else {
+		if err != nil {
```

Comment:
What error is this? does this code build? 🤔

Reply:
it works perfectly :D

Reply:
Well now that I cleaned up the rest, it does not work anymore 😝